parquet/src/main/java/org/apache/iceberg/parquet/ParquetMetricsRowGroupFilter.java

```diff
@@ -47,6 +47,7 @@
 import org.apache.parquet.schema.PrimitiveType;
 
 public class ParquetMetricsRowGroupFilter {
+
```

Comment:
Looks like this is a non-functional change to an unrelated file. Could you revert it? Same with `ParquetDictionaryRowGroupFilter`.

Reply:
Strange, I remember I fixed things like this, maybe the fix was reverted by merging.  Anyway, let me fix this.